src/features/camera.md

```diff
@@ -0,0 +1,240 @@
+[`Camera3dBundle`][bevy::Camera3dBundle]. It will simply draw all renderable
+entities that are [visible][cb::visibility].
+
+This page gives a general overview of cameras in Bevy. Also see the dedicated
+pages for [2D cameras][cb::camera-2d] and [3D cameras][cb::camera-3d].
+
+Practical advice: always create [marker components][cb::component-marker] for
+your camera entities, so that you can [query][cb::query] your cameras easily!
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:simple}}
+```
+
+## The Camera Transform
+
+Cameras have [transforms][cb::transform], which can be used to position or
+rotate the camera. This is how you move the camera around.
+
+For examples, see these [cookbook][chapter::cookbook] pages:
+ - [3D pan-orbit camera][cookbook::pan-orbit-camera], like in 3D editor apps
+
+If you are making a game, you should implement your own custom camera controls
+that feel appropriate to your game's genre and gameplay.
+
+### Zooming the camera
+
+You *could* use the transform's scale to scale/stretch the camera; however,
+don't confuse that with "zooming"! Typically, zooming should be done
+using the [projection](#projection).
+
+For orthographic projections, change the projection's scale. This way you can be
+confident about how exactly coordinate/units map to the screen, and avoid
+imprecision or visual artifacts. This also helps avoid scaling artifacts with 2D
+assets.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/d2.rs:zoom}}
+```
+
+For perspective projections, change the FOV. This achieves the desired 3D effect
+of zooming with a lens. Decrease the FOV to "zoom in" (make objects appear
+closer). Increase the FOV to "zoom out" (make objects appear further away,
+increase the stretching due to the perspective effect).
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/d3.rs:zoom}}
+```
+
+That said, in some cases (such as editor apps), using the transform scale
+might make sense. The effect it produces (stretching the content as it appears
+on screen) might feel better to control.
+
+## Projection
+
+The camera projection is responsible for mapping the coordinate system to the
+viewport. This effectively determines the "coordinate space" you are working in.
+
+Bevy provides two kinds of projections:
+[`OrthographicProjection`][bevy::OrthographicProjection] and
+[`PerspectiveProjection`][bevy::PerspectiveProjection]. They are configurable,
+to be able to serve a variety of different use cases. See the dedicated pages
+for [2D cameras][cb::camera-2d] and [3D cameras][cb::camera-3d] to learn more
+about what you can do with them.
+
+It is possible to implement your own [custom camera
+projections][cb::camera-custom-projection]. This can give you full control over
+the coordinate system. However, beware that things might behave in unexpected
+ways if you violate Bevy's [coordinate system conventions][cb::coords]!
```

Comment:
You probably have/will document it but just noting that reverse z is an important detail.